access/ms-teams/_tpl/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.11",
```

Comment:
`1.14` is the latest version
Is there a reason we are using 1.11?

Reply:
That was auto generated by the ms tool. Are you sure by 100% that it would work?

Reply:
https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema#manifestversion
Feel free to leave as 1.11 or update to 1.12